src/components/Login/Login.js

```diff
@@ -14,9 +14,23 @@ class Login extends Component {
     this.setState({[e.target.name]: e.target.value})
   }
 
-  fetchUserData = (e) => {
+  checkUserData = (e) => {
     e.preventDefault();
-    fetch('https://rancid-tomatillos.herokuapp.com/api/v1/login',
+    this.fetchUserData()
```

Comment:
So does this work because you do the initial fetch and then this.fetchUserData, followed by the .then statements? Does it work because the first fetch resolves to true, which allows this.fetchUserData to run?